openapiart/common.go

```diff
@@ -126,11 +130,137 @@ type httpClient struct {
 // All api rpcs MUST call Validate
 var validation []string
 
-func Validate() {
+func validationResult() error {
 	if len(validation) > 0 {
 		for _, item := range validation {
 			fmt.Println(item)
 		}
-		panic("validation errors")
+		validation = nil
+		return fmt.Errorf("validation errors")
```

Comment:
I think it would be better to join all validation items into a string and return that as the error as opposed to printing them out, allowing users to report the validation errors however they choose.